### PR TITLE
enable Travis / Sonar #1

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

**Comment by lhein** [comment]
@apupier addressed your suggestions

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - "sh -e /etc/init.d/xvfb start"
+  - 'export PATH=./node_modules/.bin:$PATH'
+  - 'npm install -g typescript'
+install:
+#  - 'npm install --ignore-scripts'
+  - 'npm install'
+  # https://github.com/travis-ci/travis-ci/issues/8813
+  - 'rm -f ./node_modules/.bin/which'
+#  - 'npm run vscode:prepublish'
+  - 'npm install -g vsce'
```
Comment (apupier) [inline .travis.yml]: no need to install vsce as the next line is commented

Comment (lhein) [reply .travis.yml]: There is an open issue about deployment to marketplace. I am not going to remove it now to add it back soon.

Comment (apupier) [reply .travis.yml]: it would be cleaner

Comment (lhein) [reply .travis.yml]: it makes no sense imo

**Pull request merged**